frontend/src/utils/index.js

```diff
@@ -52,7 +52,7 @@ export function emailToDisplayName (value) {
   }
 }
 
-export function serviceAccountToDisplayName (serviceAccount) {
+export function nameFromPrefixedServiceAccountName (serviceAccount) {
```

Comment:
Rename prefixedServiceAccountName everywhere to fullyQualifiedServiceAccountName